docs/jwst/photom/reference_files.rst

```diff
@@ -4,16 +4,26 @@ The photom step uses a photom reference file and a pixel area map reference
 file. The pixel area map reference file is only used when processing
 imaging and NIRSpec IFU observations.
 
-PHOTOM CRDS Selection Criteria
-------------------------------
-Selection criteria for photom reference files varies a bit from instrument
-to instrument:
+CRDS Selection Criteria
+-----------------------
 
-* FGS: Instrument and Detector
-* MIRI: Instrument, Detector, and Band
-* NIRCam: Instrument and Detector
-* NIRISS: Instrument and Detector
-* NIRSpec: Instrument and Exp_type
+PHOTOM Reference Files
+^^^^^^^^^^^^^^^^^^^^^^
+
+For FGS, photom reference files are selected based on the values of INSTRUME and DETECTOR
```

Comment:
Because ``INSTRUME`` is used as an implicit selector for all reference files, we can probably exclude it from the lists here. We don't mention ``INSTRUME`` for any of the other types.